pontoon/base/models.py

```diff
@@ -3425,3 +3432,13 @@ def calculate_stats(self, save=True):
             strings_with_warnings_diff,
             unreviewed_strings_diff,
         )
+
+
+class Comment(models.Model):
+    author = models.ForeignKey(User)
+    timestamp = models.DateTimeField(default=timezone.now)
+    translation = models.ForeignKey(Translation)
+    comment = models.TextField(blank=True)
```

Comment:
As noted above, we should rename this to `content`.

Two questions for @adngdb:
1. Should we add fields needed for Team Comments now or when we need them? I'm leaning towards the later. Anyhow, I think we could make use of your foreign key validation logic from #1503.
2. Should we make comments a separate pontoon app?